src/lib/Preview.js

```diff
@@ -814,6 +814,11 @@ class Preview extends EventEmitter {
      * @return {void}
      */
     setupUI() {
+        // Do nothing if container is already setup and in the middle of retrying
+        if (this.container && this.retryCount > 0) {
```

Comment:
This seems like a somewhat risky place for this change, since the `this.ui.setup` method is intended to destroy/replace the existing UI. Does this cover edge cases where a particular file is being retried and the user navigates to another file?

Reply:
Looking at the code for navigating prev/next files, maybe it would be better to use `retryTimeout` instead of relying on `retryCount` since I don't see that getting reset in the navigation flow.

The other approach I was thinking was to pass in a retry boolean to be more explicit when calling `load` via `handleFetchError` but I would need to propagate that value through a few functions and result in a bigger change, let me take another look